doc/changes/devel/12962.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fix displayed units in :func:`~mne.utils.misc.sizeof_fmt` to correctly use KiB, MiB, GiB, and so on, by `Clemens Brunner`_.
```

Comment:
We can only crossref things that are documented in our API reference, and `mne.utils` is not. So I would refer to the more public / end-user effect instead
```suggestion
Fix displayed units in representations of classes such as :class:`mne.io.Raw` to correctly use KiB, MiB, GiB, and so on, by `Clemens Brunner`_.
```